src/common/pa_converters.c

```diff
@@ -329,24 +330,62 @@ static const double const_1_div_2147483648_ = 1.0 / 2147483648.0; /* 32 bit mult
 
 /* -------------------------------------------------------------------------- */
 
+static inline int SetRoundingMode()
```

Comment:
From an API standpoint, it seems odd to me to have a Set function() that does not take a parameter.
Could we change SetRoundingMode() to SetRoundingModeToNearest()?

Reply:
Ok, agree. Updated PR.